sql/core/src/test/scala/org/apache/spark/sql/PlanStabilitySuite.scala

```diff
@@ -115,7 +117,7 @@ trait PlanStabilitySuite extends TPCDSBase with DisableAdaptiveExecutionSuite {
    * @param explain the full explain output; this is saved to help debug later as the simplified
    *                plan is not too useful for debugging
    */
-  private def generateApprovedPlanFile(plan: SparkPlan, name: String, explain: String): Unit = {
+  private def generateGoldenFile(plan: SparkPlan, name: String, explain: String): Unit = {
```

Comment:
Address the comment from https://github.com/apache/spark/pull/29270#discussion_r476174979